src/App.java

```diff
@@ -1,5 +1,562 @@
+        "║                                                            ║\r\n" + //
+        "║      > (1) Preset File -Recommended option-                ║\r\n" + //
+        "║      > (2) File Slot 1 (only works if you add a file)      ║\r\n" + //
+        "║      > (3) Add a file                                      ║\r\n" + //
+        "╠════════════════════════════════════════════════════════════╣\r\n");
+
+        int openingChoice= comms.numConsoleInput();
+         if(openingChoice == 1){ //If the user wants they can use the pre-existing file
+            //I added this because I got tired of typing in BookList.txt every single time the program ran
+            fileName= "BookList.txt";
+            comms.consoleOutput("Thanks for selecting a file, its contents are now being read by the program");
+            return;
+        } 
+        
+        else if(openingChoice==2){//Choice two allows user to pick their own file, this also allows them to have 2 different libraries
+            if(File1==null){//If the string is empty program returns the following
+                comms.consoleOutput("Add a file first"); 
+                addFile();//goes back to start of this method, the CUI
+                return;
+            } //nested if statement ending
+
+            else{//If the string is not null
+            fileName= File1;
+            try{io.openFile(fileName);//Checks whether the string added is actually a valid file name using try & catch
+                io.closeFile();}catch(FileNotFoundException e){
+                    comms.consoleOutput("File not found, add a file that in accessible");
+                    addFile();
+                }//nested else statement ending
+                
+            comms.consoleOutput("Thanks for selecting a file, its contents are now being read by the program");
+            
+            
+        }//else if statement ending
+    }
+
+        else if(openingChoice==3){//Allows user to add their own file
+        comms.consoleOutput("Okay, to be honest I haven't really tested this");
+        comms.consoleOutput("");
+        comms.consoleOutput("However, if you make the file in the correct format and add it to the correct place it should work");
+        comms.consoleOutput("");
+        comms.consoleOutput("Type in your file name, exactly how it is. DONT forget to add the .txt at the end");
+        File1=comms.textConsoleInput();//saves user input to the private variable
+        comms.consoleOutput("");
+        comms.consoleOutput("File option added");
+        addFile();//reruns the program with the updates value of the private variable
+        }//end else if statement
+
+
+        else{//Incase user input any integer value other than 1-3
+        comms.consoleOutput("You messed up but that's okay, pick again");
+        addFile();
+        return;
+        }//end else statement
+    }//end method
+    
+
+
+    
+    public void menu(){
+        comms.consoleOutput("\r\n" + //

```

Comment:
FYI, it should be sufficient to use "\n" rather than "\r\n" in essentially every OS. I think.